packages/extension/package.json

```diff
@@ -101,8 +102,6 @@
     "mini-css-extract-plugin": "^1.6.2",
     "nock": "^13.2.4",
     "node-fetch": "^2.6.6",
-    "sass": "^1.53.0",
```

Comment:
Not sure if the sass is even needed anymore.
But we can re-evaluate in another test case.

Reply:
Not sure as well but I agree let's revisit it later :)